lib/runctl.js

```diff
@@ -27,7 +27,12 @@ var server = require('strong-control-channel/server').create(onRequest);
 // attach targetctl to the master).
 var ipcctl = null;
 if (cluster.isMaster && process.send) {
```

Comment:
And if supervisor isn't a direct child process, it won't have process.send: docker images, for example.